include/fir/Dialect/FIROps.td

```diff
@@ -117,6 +120,7 @@ class fir_SimpleOp<string mnemonic, list<OpTrait> traits>
   }];
 }
 
+// Base builder for allocate operations
```

Comment:
Would using `let description =[{Base builder for allocate operations}]` here instead of `//` comments make sens here ?

Reply:
Following the style in place like Ops.td with this.

I think there is a fine distinction drawn between a sort of generalist "reads like a tutorial" documentation (description) and practitioners "what this code is for" comments.